src/main/java/org/spongepowered/common/world/SpongeWorldArchetypeBuilder.java

```diff
@@ -266,7 +266,7 @@ public SpongeWorldArchetypeBuilder reset() {
         this.enabled = true;
         this.loadOnStartup = true;
         this.keepSpawnLoaded = null;
-        this.generateSpawnOnLoad = this.dimensionType.getConfigAdapter().getConfig().getWorld().getGenerateSpawnOnLoad();
+        this.generateSpawnOnLoad = true; // TODO this.dimensionType.getConfigAdapter().getConfig().getWorld().getGenerateSpawnOnLoad();
```

Comment:
This is actually a fine change with no TODO needed. DimensionType specifying if I'll generate spawn on load is just a Forge feature. This call will likely be changed to do...

ModWorldHooks.shouldGenerateSpawnOnLoad(DimensionType dimensionType);